doc/whats_new/upcoming_changes/sklearn.tree/32119.fix.rst

```diff
@@ -0,0 +1,4 @@
+- Fix calculation of nodes impurity in :class:`tree.DecisionTreeRegressor`
+  and :class:`ensemble.RandomForestRegressor` when missing values are
+  present.
```

Comment:
Could you be a bit more specific when this error occured? For all criteria? Allways when missing values are present?